### ISSUE TITLE
Could't match files

### ISSUE THREAD

**Issue by ailurus1991** [rockingdingo/deepnlp#5]
在使用 TF 1.0.0，最新的 deepnlp，使用 model 是 textsum 自动摘要。

```
ERROR:tensorflow:Couldn't match files for checkpoint /mnt/python/pypi/deepnlp/deepnlp/textsum/ckpt/headline_large.ckpt-48000
```

是需要把 ckpt/ 中解压后的数据手动放进这里一份吗？

感谢

**Comment by rockingdingo** [comment]
Hi 
如果你已经解压了，这个路径是保存在文件 ./deepnlp/textsum/ckpt/checkpoint  中的，参考（https://github.com/rockingdingo/deepnlp/blob/master/deepnlp/textsum/ckpt/checkpoint），把里面的 model_checkpoint_path: "/mnt/python/pypi/deepnlp/deepnlp/textsum/ckpt/headline_large.ckpt-48000" 这一行换成你自己的安装目录  model_checkpoint_path: "your_install_path/ckpt/headline_large.ckpt-48000" 这样模型就能找到了；

原因是python文件 headline.py 中用的
'''
...
saver.restore(session, tf.train.latest_checkpoint(FLAGS.train_dir))
'''

会从train_dir中先找到 checkpoint文件，再读里面的model_checkpoint_path的路径。

**Comment by ailurus1991** [comment]
Thanks, 了解了。

**Comment by 070110053** [comment]
第一步，出现
DataLossError (see above for traceback): Unable to open table file /Users/guoguo/PycharmProjects/textsum/ckpt/headline_large.ckpt-48000.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?
	 [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_recv_save/Const_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]

**Comment by 070110053** [comment]
不知道数据怎样处理，也解压了，路径名也改了，就是不行，请帮忙看下，万分感谢！
